src/components/org/PositionCard/components/TimelineInstance.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useMemo } from 'react';
+import styles from '../styles.less';
+import { PositionInstance } from '@equinor/fusion';
+import { useTooltipRef } from '@equinor/fusion-components';
+import classNames from 'classnames';
+
+type TimelineInstanceProps = {
+    instance: PositionInstance;
+    currentInstance: PositionInstance | null;
+    calculator: (time: number) => number;
```

Comment:
The position calculator is really an internal calculation that should belong to the TimelineInstance component, not the consumer

Reply:
Moved calculator